src/Plugin/BasePayload.php

```diff
@@ -14,10 +14,14 @@
 use Manticoresearch\Buddy\Core\ManticoreSearch\Endpoint;
 use Manticoresearch\Buddy\Core\ManticoreSearch\Settings;
 use Manticoresearch\Buddy\Core\Network\Request;
+use Manticoresearch\Buddy\Core\Tool\SqlQueryParser;
 
-abstract class BasePayload {
+abstract class BasePayload
+{
```

Comment:
In our coding style, we place `{` at the end of the class name.

Reply:
As I recall, it was quite challenging to find a solution for implementation in the autofixer. However, if you have any ideas on how to expedite this process, feel free to contribute here – https://github.com/manticoresoftware/php-code-standard

Reply:
Done